test/tasks/example/sep/028-opcm-update-prestate-v410/config.toml

```
@@ -0,0 +1,27 @@
+templateName = "OPCMUpdatePrestateV410"
+
+[[l2chains]]
+chainId = 11155420
+name = "OP Sepolia Testnet"
+
+[[opcmUpgrades]]
+chainId = 11155420
+absolutePrestate = "0x03eb07101fbdeaf3f04d9fb76526362c1eea2824e4c6e970bdb19675b72e4fc9"  # op-program/v1.6.1 - https://github.com/ethereum-optimism/superchain-registry/blob/47f21fb1291999cb9a46653dd92d7cd55b6e3821/validation/standard/standard-prestates.toml#L6C9-L6C75
```

fyi - there is a typo in these prestates:
```suggestion
absolutePrestate = "0x03eb07101fbdeaf3f04d9fb76526362c1eea2824e4c6e970bdb19675b72e4fc8"  # op-program/v1.6.1 - https://github.com/ethereum-optimism/superchain-registry/blob/47f21fb1291999cb9a46653dd92d7cd55b6e3821/validation/standard/standard-prestates.toml#L6C9-L6C75
```

---

Hey @mbaxter this was intended, the plan here is to prepare the tasks ahead of time and update them with the correct Fusaka Prestate as soon as it is available. 

All Fusaka Prestate tasks have the same placeholder/wrong prestate`0x03eb07101fbdeaf3f04d9fb76526362c1eea2824e4c6e970bdb19675b72e4fc9`. The typo is intended because the correct prestate (ending with `8`) is already used by these chains and the task would not simulate.